src/clj/slf4j_timbre/adapter.clj

```diff
@@ -0,0 +1,37 @@
+(ns slf4j-timbre.adapter
+	(:gen-class
+		:name com.github.fzakaria.slf4j.timbre.TimbreLoggerAdapter
+		:extends org.slf4j.helpers.MarkerIgnoringBase)
+	(:require [taoensso.timbre :as timbre])
+	(:import [org.slf4j.helpers FormattingTuple MarkerIgnoringBase MessageFormatter]))
+
+(defmacro ^:private wrap
```

Comment:
i don't macros much. Just curious why this is written as one versus just a wrapper that takes a function as a parameter